pkg/istioversions/versions.yaml

```diff
@@ -124,7 +130,9 @@ versions:
       - https://istio-release.storage.googleapis.com/charts/gateway-1.21.6.tgz
       - https://istio-release.storage.googleapis.com/charts/cni-1.21.6.tgz
       - https://istio-release.storage.googleapis.com/charts/ztunnel-1.21.6.tgz
-  - name: latest
+  - name: master
+    ref: 1.25-alpha.c2ac935c588899579e1cee38b205e4e6824a4638
```

Comment:
Why does `ref` point to the version number instead of the version name? Isn't it more natural for the reference to point to another object by it's name rather than one of its other properties? It would also make the code simpler.